src/core/mac/sub_mac.hpp

```diff
@@ -389,6 +392,26 @@ class SubMac : public InstanceLocator
      */
     void SetCslChannel(uint8_t aChannel);
 
+    /**
+     * This method indicates if CSL channel has been explicitly specified by the upper layer.
+     *
+     * In other words, this indicates that if the PIB attribute `macCSLChannelMask` is 0. In current implementation,
+     * it's allowed to set this to 0 again.
```

Comment:
`mIsCslChannelSpecified` is only related to `mCslChannel`, and it is `mCslChannel` that is required to be a valid channel value within `macCSLChannelMask`. 
Why brings the new `macCSLChannelMask` concept, which is not implemented or well documented, when introducing `mIsCslChannelSpecified` related APIs?

For simplicity, or at least before `macCSLChannelMask` is introduced, I think it should be enough to only specifically document that `0` is to clear the Csl Channel specified.

Thoughts?

Reply:
SGTM.